Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
Yep, that works a lot better. Maybe close this and commit that separately? There are 2 `push` that need to be replaced in lines `35` & `48` (for comments starting with `--` and `//`)